app/models/hold.rb

```diff
@@ -38,4 +38,32 @@ def grad_date
   def author_names
     self.thesis.users.map { |u| u.name }.join("; ")
   end
+
+  def created_by
+    if self.versions.first.event == 'create'
+      creator_id = self.versions.first.whodunnit
+      user = User.find_by(id: creator_id)
+      user.kerberos_id
```

Comment:
I think it would be good to protect against future staff User deletion here and maybe to a check that `user` exists before and if not displaying something like `User no longer active: user_id XYZ`. [Something more terse would be great but that's the idea of what we likely need to convey for this edge case to prevent error states]